src/components/ReportActionItem/TaskView.js

```diff
@@ -69,7 +71,7 @@ function TaskView(props) {
                         accessibilityLabel={taskTitle || props.translate('task.task')}
                     >
                         {({pressed}) => (
-                            <>
```

Comment:
Let's wrap the top level view in OfflineWithFeedback and pass to it only the errors prop coming from `report.errorFields.editTask`, and an onClose callback to clear the error message and pendingFields. That way if there's an unexpected error on the back end the user will see an error message which they can dismiss.

I can help test this.

https://github.com/Expensify/App/blob/0965cb31f3fd4ff5534b5bcefa11d974b2dd4c81/src/components/ReportActionItem/TaskView.js#L56-L57